### PR TITLE
Feature/pic 3566 persist filters

### PR DESCRIPTION
**Pull request by ArronChave3**

### CODE REVIEW

File: server/services/case-service.js [hunk 1]
```diff
@@ -91,6 +91,7 @@ const createCaseService = apiUrl => {
       }
 
       const response = await request(apiUrlBuilder.href)
+      console.log('🚀 ~ createCaseService ~ response:', response)
```
Comment (johnyfarrar) [inline server/services/case-service.js:94]: Might need to remove this....

**Review by johnyfarrar — CHANGES_REQUESTED** [review]
One quick change but the tests are failing too...

**Review by sameermoj — DISMISSED** [review]

File: public/src/javascripts/filter-component.js [hunk 2]
```diff
@@ -109,12 +109,39 @@
...
+  function configureClearTags() {
+    function clearFilter(e) {
       e.stopPropagation()
       const checkbox = document.getElementById(e.target.dataset['controls'])
       checkbox.click()
-      document.getElementById('pac-filter-form').submit()
+      const form = document.getElementById('pac-filter-form')
+      var formData = new FormData(form);
+      // output as an object
+      console.log(Object.fromEntries(formData));
```
Comment (johnyfarrar) [inline public/src/javascripts/filter-component.js:140]: Just got this one to remove

File: public/src/javascripts/filter-component.js [hunk 3]
```diff
@@ -149,8 +176,9 @@
     })
   }
 
-  function configureResetButton () {
-    function resetFilters (e) {
+  function configureResetButton() {
+    function resetFilters(e) {
+      console.log('Reset filters button clicked')
```
Comment (johnyfarrar) [inline public/src/javascripts/filter-component.js:181]: Another one...

**Review by johnyfarrar — CHANGES_REQUESTED** [review]
Just removing a couple of console logs

**Review by johnyfarrar — APPROVED** [review]
Great work

**Pull request merged**